Navigation restructuring (#403)
codatio/codat-docs: docs/configure/products.md

---
title: "Enable products"
sidebar_label: "Products"
description: "Enable additional products you need for your usecase"
---

import Products from "@components/global/Products";

<p>
  To build integrations for small business lending, try our <a href="/assess/overview">Assess</a>{" "}
  product. If you're developing accounting integrations for a PoS or eCommerce
  platform, go to <a href="/sfc/overview">Sync for Commerce</a>. For accounting integrations for corporate cards or
  expense management platforms, refer to <a href="/sync-for-expenses/overview">Sync for Expenses</a>. If you plan to push 
  your customers' bank transaction data into their accounting platforms, check out our <a href="/bank-feeds-api/overview">Bank Feeds API</a>.
</p>

<p>
  Building something else? Explore our <a href="/accounting-api/overview">Accounting API</a>,{" "}
  <a href="/banking-api/overview">Banking API</a>, and <a href="/commerce-api/overview">Commerce API</a> to pull, push, transform, and
  visualize data from 35+ financial data sources, integrated with a single data model.
</p>

## Enabling products

> By default you'll have access to our standardized integration products - our Accounting, Banking, and Commerce APIs. For our other products, you can either enable then as you sign up, or [pmanage them in the Portal](https://app.codat.io/settings/products) (admins only).

Reviewer: ```suggestion
By default, you'll have access to our standardized integration products - our Accounting, Banking, and Commerce APIs. For our other products, you can either enable them as you sign up or [manage them in the Portal](https://app.codat.io/settings/products) (users with Administrator roles only).
```
